docs/interpreter/jdbc.md

```diff
@@ -221,6 +222,66 @@ SELECT name, country, performer
 FROM demo.performers
 WHERE name='{{"{{performer=Sheryl Crow|Doof|Fanfarlo|Los Paranoia"}}}}'
 ```
+### Usage `precode`
+You can set `precode` for each data source. Code runs once while opening the connection.
+
```

Comment:
you might want to add a line here to say `for example, first setup properties to mysql...`